src/components/slices/project-grid.module.scss

```diff
@@ -24,38 +52,81 @@
   position: relative;
   width: 743px;
   max-width: 100%;
-  aspect-ratio: 10/7;
+  aspect-ratio: 1/1;
   border-radius: $bradius;
   overflow: hidden;
   display: flex;
-  background: $color-darkgreen;
+  border: 10px solid $color-white;
+
   .image {
-    top: 50%;
-    left: 50%;
-    transform: translate(-50%, -50%);
-    position: absolute;
+    top: 0;
+    left: 0;
     width: 100%;
     height: 100%;
     transition: all 0.2s ease-in-out;
   }
-  &:hover .image {
-    width: 110%;
-    height: 110%;
+
+  .overlay {
+    display: none;
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    background-color: rgba($color-transparentwhite, 0.8);
```

Comment:
Would boost this to .9 honestly, you'll see in the screenshot at the end of this review that it's' still a little hard to read on some photos.